internal/database/mariadb/component_instance_test.go

```diff
@@ -1643,6 +1687,66 @@ var _ = Describe("ComponentInstance - ", Label("database", "ComponentInstance"),
 			})
 		})
 	})
+	When("Creating a self-referencing ComponentInstance", Label("CreateSelfReferencingComponentInstance"), func() {
```

Comment:
Is this a duplicated test? Double check with
It("can insert correctly (without ParentID)"...
and
It("can insert correctly (with ParentID)" ...